pkg/controller/dataimportcron-controller_test.go

```diff
@@ -353,7 +354,8 @@ func createDataImportCronReconciler(objects ...runtime.Object) *DataImportCronRe
 
 func newDataImportCronWithImageStream(name string) *cdiv1.DataImportCron {
```

Comment:
Can we add a test that tests the permutations of imagestreams, and invalid ones?
imagestream
imagestream:
imagestream:tag
imagestream:tag:label

Reply:
added